index.html

```diff
@@ -95,16 +95,16 @@ <h2 class="about">About myself</h2>
       <div class="formsection">
         <h3 class="styleh3">CONTACT US</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" class="formstyle">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email">
-          <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqrojze" method="POST" class="formstyle">
+          <input type="text" placeholder="Text input" maxlength="30" required>
+          <input type="email" placeholder="email" required>
+          <textarea name="text" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+          <button type="submit" class="button_submit"><span>Submit</span></button>
```

Comment:
Hello team ✋

- [ ] Just like the previous reviewer pointed out, you may please implement the button interactions properly. I can't see the hover effect. Please implement the hover and enable effects accordingly